worker_generic/manager/__main__.py

```diff
@@ -0,0 +1,59 @@
+from manager.load_env import load_env
+from manager.load_env import load_file_or_env
+import mysql.connector
+from manager.cryptography import decrypt_json
+import os
+import subprocess
+
+print("Initializing worker environment")
+
+CONNECTORS_PATH = load_env("CONNECTORS_PATH", "./../")
+
+MYSQL_HOST = load_env("MYSQL_HOST", "localhost")
+MYSQL_PORT = load_env("MYSQL_PORT", 3306)
+MYSQL_USER = load_env("MYSQL_USER", "root")
+MYSQL_PASSWORD = load_file_or_env("MYSQL_PASSWORD")
+MYSQL_DATABASE = load_env("MYSQL_DATABASE", "openpodcast_auth")
+
+OPENPODCAST_ENCRYPTION_KEY = load_file_or_env("OPENPODCAST_ENCRYPTION_KEY")
+if not OPENPODCAST_ENCRYPTION_KEY:
+    print("No OPENPODCAST_ENCRYPTION_KEY found")
+    exit(1)    
+
+# try to connect to mysql or exit otherwise
+try:
+    db = mysql.connector.connect(
+        host=MYSQL_HOST,
+        port=MYSQL_PORT,
+        user=MYSQL_USER,
+        passwd=MYSQL_PASSWORD,
+        database=MYSQL_DATABASE,
+    )
+except Exception as e:
+    print("Error connecting to mysql: ", e)
+    exit(1)
```

Comment:
It's generally good practice to avoid catch-all exceptions. (For instance if the `connect` blocks and never finishes, the program cannot be terminated with `Ctrl+C` right now.

Catching `except mysql.connector.Error` is enough:

```suggestion
try:
    db = mysql.connector.connect(
        host=MYSQL_HOST,
        port=MYSQL_PORT,
        user=MYSQL_USER,
        passwd=MYSQL_PASSWORD,
        database=MYSQL_DATABASE,
    )
except mysql.connector.Error as e:
    print("Error connecting to mysql: ", e)
    exit(1)
```